src/data/petsData.ts

```diff
@@ -140,25 +146,101 @@ export const petsData: PetProps[] = [
       "/images/fromLove/tola/tola01.jpg",
       "/images/fromLove/tola/tola02.jpg",
       "/images/fromLove/tola/tola03.jpg",
-      "/images/fromLove/tola/tola04.jpg",
       "/images/fromLove/tola/tola05.jpg",
       "/images/fromLove/tola/tola06.jpg",
       "/images/fromLove/tola/tola07.jpg",
       "/images/fromLove/tola/tola08.jpg",
       "/images/fromLove/tola/tola09.jpg",
-      "/images/fromLove/tola/tola10.jpg",
       "/images/fromLove/tola/tola11.jpg",
       "/images/fromLove/tola/tola12.jpg",
       "/images/fromLove/tola/tola13.jpg",
-      "/images/fromLove/tola/tola14.jpg",
       "/images/fromLove/tola/tola15.jpg",
     ],
   },
+  {
+    name: "bestia",
+    declension: "bestii",
+    description:
+      "Nasz kotek, znaleziony na placu budowy, jest z nami od kwietnia 2017 roku. Bestia jest wysterylizowany, a w domu – niekwestionowany przywódca, którego słuchają się wszystkie psy.",
+    gender: "male",
+    images: [
+      "/images/fromLove/bestia/bestia01.jpg",
+      "/images/fromLove/bestia/bestia02.jpg",
+      "/images/fromLove/bestia/bestia03.jpg",
+      "/images/fromLove/bestia/bestia04.jpg",
+      "/images/fromLove/bestia/bestia05.jpg",
+      "/images/fromLove/bestia/bestia06.jpg",
+      "/images/fromLove/bestia/bestia07.jpg",
+      "/images/fromLove/bestia/bestia08.jpg",
+      "/images/fromLove/bestia/bestia09.jpg",
+      "/images/fromLove/bestia/bestia10.jpg",
+      "/images/fromLove/bestia/bestia11.jpg",
+      "/images/fromLove/bestia/bestia12.jpg",
+      "/images/fromLove/bestia/bestia13.jpg",
+      "/images/fromLove/bestia/bestia14.jpg",
+      "/images/fromLove/bestia/bestia15.jpg",
+    ],
+  },
+  {
+    name: "boni",
+    declension: "boni",
+    description:
+      "Poznajcie Boni – uroczego hawańczyka, który od razu skradł nasze serca! Boni to spełnienie marzeń naszych synów – kochany, ciepły i zawsze gotowy na pieszczoty. Jest z nami od 28 stycznia 2024 roku i już stał się ukochaną, żywą przytulanką całej rodziny.",
+    gender: "male",
+    images: [
+      "/images/fromLove/boni/boni01.jpg",
+      "/images/fromLove/boni/boni02.jpg",
+      "/images/fromLove/boni/boni03.jpg",
+      "/images/fromLove/boni/boni04.jpg",
+      "/images/fromLove/boni/boni05.jpg",
+      "/images/fromLove/boni/boni06.jpg",
+      "/images/fromLove/boni/boni07.jpg",
+      "/images/fromLove/boni/boni08.jpg",
+      "/images/fromLove/boni/boni09.jpg",
+      "/images/fromLove/boni/boni10.jpg",
+      "/images/fromLove/boni/boni11.jpg",
+      "/images/fromLove/boni/boni12.jpg",
+      "/images/fromLove/boni/boni13.jpg",
+      "/images/fromLove/boni/boni14.jpg",
+      "/images/fromLove/boni/boni15.jpg",
+      "/images/fromLove/boni/boni16.jpg",
+      "/images/fromLove/boni/boni17.jpg",
+      "/images/fromLove/boni/boni18.jpg",
+      "/images/fromLove/boni/boni19.jpg",
+      "/images/fromLove/boni/boni20.jpg",
+      "/images/fromLove/boni/boni21.jpg",
+    ],
+  },
+  {
+    name: "grażynka",
```

Comment:
@Delicadency  You need to use the name without the Polish sign, possibly change the routing, with grażynka = grazynka because with the Polish sign routing does not work!